packages/story-editor/src/index.js

```diff
@@ -17,10 +17,13 @@
 /**
  * Internal dependencies
  */
-import App from './editorApp';
+import { default as StoryEditor } from './editorApp';
+import PostLock from './components/postLock';
+import StatusCheck from './components/statusCheck';
```

Comment:
Just noting that we might consider just moving these to `wp-story-editor` in the future as these could be considered WP-specific.